i18n/enu/out/project/saveProject.i18n.json

```diff
@@ -1,4 +1,4 @@
 {
-  "autolispext.project.saveproject.noprojecttosave": "No project to save.",
-  "autolispext.project.saveproject.generateprjcontentfailed": "Failed to generate project content."
+	"autolispext.project.saveproject.noprojecttosave": "No project to save.",
```

Comment:
are these changes not necessary?

Reply:
> are these changes not necessary?

It is necessary, because once we replace it with the entire file from localization team, we won't have t this diff anymore in the future.

Reply:
You can turn on the following option to ignore them:
![image](https://user-images.githubusercontent.com/57521938/119755550-c60cbd80-bed4-11eb-96d0-9895a1bfadc8.png)